src/Microsoft.Azure.WebJobs.Host/Blobs/BlobPath.cs

```diff
@@ -55,16 +55,39 @@ public static BlobPath ParseAndValidate(string value, bool isContainerBinding =
             return path;
         }
 
-        public static BlobPath Parse(string value, bool isContainerBinding)
+        public static string TryConvertAbsUrlToContainerBlob(string value)
         {
-            BlobPath path;
+            try
+            {
+                Uri uri = new Uri(value); // default to UriKind.Absolute
+                return uri.LocalPath.Substring(1); // [0] is slash, same logic as ToBlobPath() in StorageAnalyticsLogEntry.cs
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
 
-            if (!TryParse(value, isContainerBinding, out path))
+        // Does not valid containerName and blobName
+        public static void ExtractContainerNameAndBlobName(string src, out string containerName, out string blobName)
+        {
+            if (String.IsNullOrEmpty(src))
             {
-                throw new FormatException($"Invalid blob path '{value}'. Paths must be in the format 'container/blob'.");
+                throw new FormatException($"can not get container name and blob name from {src}");
             }
+            int slashIndex = src.IndexOf('/');
```

Comment:
we already use our own way of retrieving containerName + blobName from blobUrl
https://github.com/Azure/azure-webjobs-sdk/blob/e692a1ff0db6decf1ba216752c7d86414d4489ca/src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/StorageAnalyticsLogEntry.cs#L134-L135
https://github.com/Azure/azure-webjobs-sdk/blob/26e3e7aaca039d9676f5aca41e7bb088619bb68a/src/Microsoft.Azure.WebJobs.Host/Blobs/BlobPath.cs#L92-L93

I can either call the existing method, or change it to

`var blob = new CloudBlob(new Uri(blobUrl))`
`containerName = blob.Container.Name`
`blobName = blob.Name`